web/src/App.js

```diff
@@ -14,8 +13,7 @@ class App extends Component {
       <div className="App">
         <Switch>
           <header className="App-header">
-            <Route path='/' exact>
-              {/* <Header data={data.main}/> */}
+            <Route path='/player/registration' exact>
```

Comment:
This is a route for the front end. So it doesn't need to align with the back-end route. 

If we set the home route like this, the visitors should enter "softballberlin.com/player/registration", which is unnecessarily long.

using `/` would be better in my opinion for the front end.